exporter/cortexexporter/README.md

```diff
@@ -0,0 +1,15 @@
+Exporter to a Cortex instance, or any Prometheus Remote Write compatible backend.
+
+_here is a link to the overall project [design](https://github.com/open-telemetry/opentelemetry-collector/pull/1464)_
+
+File structure:
+
+- cortex.go: exporter implementation. Converts and sends OTLP metrics
+
+- helper.go: helper functions that cortex.go uses. Performs tasks such as sanitizing label and generating signature string
+
+- config.go: configuration struct of the exporter
+
+- factory.go: initialization methods for creating default configuration and the exporter
+
+- tetsutil.go: helper methods for generating OTLP metrics and Cortex TimeSeries for testing.
```

Comment:
Some bullets have periods while others don't. I think using using code formatting for the file names and capitalizing the first letter of the description can help readability here.